api/src/main/resources/client_api.yaml

```diff
@@ -179,7 +186,33 @@ definitions:
       orderBy:
         description: >
           An array of columns to sort the resulting data by, taken from the table specified above,
-          each one optionally followed by " DESC" for descending sort order. (Default varies by table.)
+          each one optionally enclosed in "DESCENDING()" for descending sort order.
+          (Default sort order is person_id followed by the ID of the specified table.)
+        type: array
+        items:
+          type: string
+
+  AnnotationQuery:
+    type: object
+    description: >
+      A specification for retrieving annotation values or review statuses entered for participants.
+    properties:
+      annotations:
```

Comment:
should this be named something else since it can be an annotation name, review status, or person id?

Reply:
Renamed to "columns".